application-templates/typescript/assets/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "assets-typescript",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite --port 3000",
+    "start": "serve public -l 8080 --cors",
+    "test": "jest --detectOpenHandles",
```

Comment:
This is inconsistent with the javascript project. Should we add tests for this template?

cc: @prateek-ct